tinyagentos/routes/project_invites.py

```
@@ -876,6 +904,11 @@ async def redeem_invite(request: Request, body: RedeemInviteIn):
                 project_id=invite["project_id"],
             )
         except Exception as exc:  # noqa: BLE001 - surface as JSON error
+            await store.rollback_to_pending(body.invite_id)
+            try:
+                await auth_store.set_decision(record['id'], 'refused', decided_by='system:invite-rollback')
+            except Exception:  # noqa: BLE001 - cleanup best-effort
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)